### PR TITLE
Add props to force fallback menu

### LINKED ISSUE #390: Add props to force fallback menu
issue is here https://github.com/comicrelief/giftaid-react/issues/230

### PR DESCRIPTION
**Pull request by LouiaR**
fixes #390

### CODE REVIEW

**Comment by LouiaR** [comment]
> Looking good man, but could we change the prop to `forceFallback` rather than `noEndpoint`? Makes it clearer what the outcome is.

Sure

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ Footer.propTypes = {
 Footer.defaultProps = {
   noLinks: false,
   noSocial: false,
-  noEndpoint: true,
+  forceFallback: true,
```
Comment (AndyEPhipps) [inline src/components/Footer/Footer.js:59]: Sorry man, I missed this- probably best that this defaults to FALSE; we don't want to make a sweeping change like this to existing products using the footer.

Comment (LouiaR) [reply src/components/Footer/Footer.js]: Done

**Comment by AndyEPhipps** [comment]
lol already approved man

**Comment by LouiaR** [comment]
> lol already approved man

Yeah merge option took while to show up I thought it may require 2 people

**Pull request merged**